Assets/Scripts/EOSSessionsManager.cs

```diff
@@ -458,14 +494,21 @@ public class EOSSessionsManager : IEOSSubManager
 
         /// <summary>
         /// After a successful <see cref="RefreshSession(string)"/> and <see cref="OnRefreshSessionFindSessionsCompleteCallback(ref SessionSearchFindCallbackInfo)"/> callback call,
-        /// this is run to inform UI of the updated Session information.
+        /// this is run to inform UI of the updated Session information. This should be subscribed to by your program's UI.
         /// </summary>
         public Action<Session, SessionDetails> UIOnSessionRefresh { get; set; }
 
         #endregion
 
         #region Manager Set Up And Utility
 
+        /* This region contains meta-management for this Manager class.
```

Comment:
For multi-line block comments, always start the block with an empty comment line, and end with an empty comment line:

```csharp
/*
 * REGION NOTES:
 * 
 * This would be an example (the header of "REGION NOTES" isn't prescriptive, 
 * but can be a useful way to indicate the intent of the comment block that is
 * not otherwise adjacent to a function or class definition).
 */
```

Reply:
Sure, that's a simple pattern to implement. I'll put that in all the regional comments.